rocketchat/aws-sns.js

```diff
@@ -0,0 +1,41 @@
+/* exported Script */
+/* globals console, _, s */
+
+/** Global Helpers
+ *
+ * console - A normal console instance
+ * _       - An underscore instance
+ * s       - An underscore string instance
+ */
+// Despite the name (SNS), intended use is for Elastic Beanstalk notifications
+// If we ever use SNS for anything else, rename.
+// eslint-disable-next-line no-unused-vars
+class Script {
+  // eslint-disable-next-line camelcase, class-methods-use-this
+  process_incoming_request({ request }) {
+    // console is a global helper to improve debug
+    // eslint-disable-next-line no-console
+    console.log(request.content_raw);
+
+    const content = JSON.parse(request.content_raw);
+
+    let text = content.Message;
+
+    if ("SubscribeURL" in content) {
+      text = `${text}\n*Subscribe URL:* ${content.SubscribeURL}`;
+    }
+
+    return {
+      content: {
+        text,
+      },
+    };
+
+    // return {
```

Comment:
Because it's based on the [Rocket.Chat template](https://rocketchat.graduway.com/admin/integrations/webhook-incoming), and can be extended for future use.